### PR TITLE
1968420: improve description of rhsm.conf format

### PR DESCRIPTION
**Pull request by ptoscano**
Add a brief description of the format of the file, mentioning a couple
of known limitations.

### CODE REVIEW

**Comment by jirihnidek** [comment]
Some native English speaker should also check this PR @tlhmerry9 @cnsnyder @wottop

**Review by jirihnidek — APPROVED** [review]
ACK

File: man/rhsm.conf.5 [hunk 1]
```diff
@@ -32,6 +32,8 @@ rhsm.conf \- Configuration file for the subscription\-manager tooling
 .SH "DESCRIPTION"
 .sp
 The rhsm\&.conf file is the configuration file for various subscription manager tooling\&. This includes \fBsubscription\-manager\fR, \fBsubscription\-manager\-gui\fR, \fBrhsmcertd\fR, and \fBvirt\-who\fR\&.
+.sp
+The format of this file is a simple INI-like structure, with pairs of \fBkey=value\fR pairs inside \fB[some_name]\fR sections. Duplicated keys in sections are not allowed, and only the last occurrence is actually used. Duplicated section names are not allowed.
```
Comment (tlhmerry9) [inline man/rhsm.conf.5:36]: My only question is that it reads odd "with pairs of key=value pairs inside of [some_name] sections.

Comment (ptoscano) [reply man/rhsm.conf.5:36]: definitely there is a "pairs" too much (the second); i'll try to rework it somehow -- I didn't want to describe again the INI format as a whole

**Comment by ptoscano** [comment]
> That makes more sense to me. :)

Thanks! Should we merge this, then?

**Pull request merged**